cmd/proxy/main.go

```diff
@@ -144,62 +135,16 @@ func main() {
 	logger.Info("Configuring router")
 	manager.ConfigureRouter(r)
 
-	cmd := exec.Command("sh", "cert.sh")
-	err = cmd.Run()
-	if err != nil {
-		log.Fatal("cert.sh: ", err)
-	}
+	logger.Infof("Serving at :%d", config.Port)
+	addr := fmt.Sprintf(":%d", config.Port)
 
-	logger.Infof("Serving at :%d", port)
-	addr := fmt.Sprintf(":%d", port)
-	err = http.ListenAndServeTLS(addr, "xudproxy.crt", "xudproxy.key", r)
-	if err != nil {
-		log.Fatal("ListenAndServe: ", err)
-	}
-	//err = http.ListenAndServe(addr, r)
-	//if err != nil {
-	//	log.Fatal(err)
-	//}
-}
+	certFile := path.Join(homedir.Get(), ".proxy", "tls.crt")
+	keyFile := path.Join(homedir.Get(), ".proxy", "tls.key")
 
-func openConsole(service string, server *socketio.Server, logger *logrus.Logger, manager *Manager) {
-	s, err := manager.GetService(service)
-	if err != nil {
-		log.Fatal(err)
-	}
-	ss, ok := s.(*xud.XudService)
-	if !ok {
-		log.Fatal("Failed to convert to SingleContainerService")
-	}
-	c, err := ss.SingleContainerService.GetContainer()
-	if err != nil {
-		log.Fatal(err)
-	}
-	execId, reader, writer, err := c.ExecInteractive([]string{"bash"})
+
+	err = http.ListenAndServeTLS(addr, certFile, keyFile, r)
+	//err = http.ListenAndServe(addr, r)
```

Comment:
nit: remove?

Reply:
@erkarl Agree. And we should make this TLS feature optional.